user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,133 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Registering a Catalog item
+
+Below you can find detailed information on how you can register a private Catalog item, either [using the Catalog UI](#using-the-catalog-ui), or [using the Catalog API](#using-the-catalog-api).
+
+## Using the Catalog UI
+
+If you use the UI, you can only register a new Catalog item, but not a new version. To register a new version for an existing item, [use the API](#using-the-catalog-api).
+
+To register a new Catalog item:
+
+1. Go to the [Catalog](https://catalog.dataminer.services).
+
+1. Open the user menu by clicking the user icon in the top-right corner.
+
+1. Click the *Register item* button.
+
+1. Select the organization for which you want to register the Catalog item.
+
+   The selection box is automatically populated with all the organizations you have access to on the dataminer.services platform.
+
+1. Select the type of the Catalog item.
+
+   The selected [Catalog item type](xref:About_the_Catalog_module#supported-catalog-item-types) defines how the Catalog item will be shown to the users. Users can also filter on type to quickly find what they are looking for when browsing the Catalog.
+
+1. Enter the name of the item.
+
+   This name will be used to display the item in the Catalog. Make sure you pick a human-readable name that makes sense.
+
+1. Once all information has been filled in, click *Register*.
+
+   This will conclude the registration process for the Catalog item and return the unique identifier for the item in question. This identifier will allow you to register versions for the Catalog item using the [API](#using-the-catalog-api).
+
+## Using the Catalog API
+
+### Authentication
+
+The API call is authenticated using [organization keys](xref:Managing_DCP_keys#organization-keys). The key needs to be added to the HTTP request in a header called **Ocp-Apim-Subscription-Key**.
+
+### API call signature
+
+The call to register a Catalog item can be found at the following URL: <https://api.dataminer.serivces/api/key-catalog/v1-0/catalog/register>. To call the API you will need to use the **POST** HTTP method.
+
+The call accepts a binary file as input. This file should be a zip file containing a ["manifest.yml"](xref:Register_Catalog_Item#manifest-file) and the item you want to upload.
```

Comment:
best to mention that it needs to be Content/Type "multipart/form-data" of type file where key is "file" and value the zip file.